src/schemas/json/apple-app-site-association.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Apple Universal Link, App Site Association",
+    "type": "object",
+    "properties": {
+        "applinks": {
+            "type": "object",
+            "properties": {
+                "apps": {
+                    "type": "array",
```

Comment:
If the value must always be an empty array, why not replace `"type"` with `"enum": [[]]`? Then you get correct completion and validation in supporting editors